### PR TITLE
Add ACE Explosives

### PR DESCRIPTION
**Pull request by DerivenLoki**
**When merged this pull request will:**
- Modifies the `Mine.base` prefabs `MineDamageManager` component, rendering gunfire unable to trigger the Trigger component
- Adds the new `ACE_Explosives_DeminingUserAction` to the `Mine.base` Prefab to allow the use of a Shovel to disable Landmines
- Modifies the existing `ACE_Chopping_UserAction` by adding a Parent Class - `ACE_ShovelDestroyUserAction`, found in ACE Core > Scripts > Game > UserActions - of which `ACE_Chopping_UserAction` now inherits from

### CODE REVIEW

File: addons/explosives/addon.gproj [hunk 1]
```diff
@@ -0,0 +1,8 @@
+GameProject {
+ ID "ACEExplosives"
```
Comment (Kexanone) [inline addons/explosives/addon.gproj:2]:
```suggestion
 ID "ACE_Explosives"
```

File: addons/core/scripts/Game/ACE_Core/UserActions/ACE_ShovelDestroyUserAction.c [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+					
+		if (!SCR_CampaignBuildingGadgetToolComponent.Cast(m_GadgetManager.GetHeldGadgetComponent()))
+			return false;
+		
+		return true;
+	}
+	
+	//------------------------------------------------------------------------------------------------
+	//! For entities that have no RplComponent, only local scripts will work
+	override bool HasLocalEffectOnlyScript()
```
Comment (Kexanone) [inline addons/core/scripts/Game/ACE_Core/UserActions/ACE_ShovelDestroyUserAction.c:119]: Remove this. We only want local effect for chopping.

File: addons/explosives/scripts/Game/ACE_Explosives/Demining/UserActions/ACE_Explosives_DeminingUserAction.c [hunk 3]
```diff
@@ -0,0 +1,12 @@
+//------------------------------------------------------------------------------------------------
+//! Mine deletion user action
+class ACE_Explosives_DeminingUserAction : ACE_ShovelDestroyUserAction
+{
+
```
Comment (Kexanone) [inline addons/explosives/scripts/Game/ACE_Explosives/Demining/UserActions/ACE_Explosives_DeminingUserAction.c:5]: Remove blank line

**Review by Kexanone — CHANGES_REQUESTED** [review]

**Review by veteran29 — COMMENTED** [review]
LGTM, I will let Kex approve

**Review by Kexanone — APPROVED** [review]

**Pull request merged**